src/website/app/demos/Icon/components/IconsByCategory.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import * as Icons from 'mineral-ui-icons';
+import { createStyledComponent } from '../../../../../styles';
+import Figure from '../components/Figure';
+import FigCaption from '../components/FigCaption';
+import FigContainer from '../components/FigContainer';
+import Heading from '../../../Heading';
+
+const CategoryHeading = createStyledComponent(Heading, ({ theme }) => ({
+  borderTop: `1px solid ${theme.borderColor}`,
+  paddingTop: theme.space_stack_lg,
+  position: 'relative',
+  textTransform: 'capitalize'
+}));
+
+const componentsByCategory: {
+  [string]: Array<React$Component<*, *, *>>
+} = Object.values(Icons)
+  .sort()
+  .reduce((acc, component) => {
+    // $FlowFixMe - Flow bug: https://github.com/facebook/flow/issues/2221
+    const { category } = component;
+    acc[category] = (acc[category] || []).concat(component);
+    return acc;
+  }, {});
+
+export default function ComponentsByCategory() {
+  return (
+    <div>
+      {Object.entries(
+        componentsByCategory
+      ).map(([category, components], index) => {
+        return (
+          <div key={index}>
+            <CategoryHeading level={4} as="h5" id={`category-${category}`}>
```

Comment:
This looks like the same code, but the headings are no longer linkable on this branch. I'm not sure what broke, since you're providing the `id` here.

Reply:
Another good catch!  Kyle updated the template during the website refactor.  I'll pull in those changes.

https://github.com/mineral-ui/mineral-ui/commit/6713f5354d0e2e9b6c0340abe8d7d1884a2899a5#diff-fe5c55fae7a9c36dda4bc23d51de4710

Reply:
ah, good. I wondered if something was lost in translation